src/Streamly/Internal/System/Command.hs

```diff
@@ -63,12 +63,16 @@ import Data.Char (isSpace)
 import Data.Word (Word8)
 import Streamly.Data.Array.Unboxed (Array)
 import Streamly.Data.Fold (Fold)
+import Streamly.Data.Stream (Stream)
+import Streamly.Data.Stream.Concurrent (MonadAsync)
 import Streamly.Internal.Data.Parser (Parser)
-import Streamly.Prelude (MonadAsync, SerialT)
 
-import qualified Streamly.Internal.Data.Fold as Fold
-import qualified Streamly.Internal.Data.Parser as Parser
-import qualified Streamly.Internal.Data.Stream.IsStream as Stream
+import qualified Streamly.Data.Fold as Fold
+import qualified Streamly.Data.Parser as Parser
+
+import qualified Streamly.Data.Stream as Stream
+import qualified Streamly.Internal.Data.Stream as Stream (parseMany)
```

Comment:
Why isn't parseMany exported from exposed modules? We should export it if it is not already.